build-tests/x86/centos/test-image-live-disk-v7/appliance.kiwi

```diff
@@ -25,20 +25,20 @@
     </preferences>
     <preferences profiles="Live">
         <type image="iso" flags="overlay" firmware="uefi" hybridpersistent_filesystem="ext4" hybridpersistent="true" kernelcmdline="console=ttyS0">
-            <bootloader name="grub2" console="serial gfxterm"/>
+            <bootloader name="grub2" console="serial"/>
```

Comment:
See, this is a ISO with the bootloader entry.

Reply:
yep got it, I did the changes by substitution, so description with no bootloader setup felt of my radar